server/app.ts

```diff
@@ -109,6 +109,27 @@ new MathigonStudioApp()
       translationsJSON
     })
   })
+  .get('/eula', (req, res) => {
+    if (!req.user) return res.redirect('/signin');
+
+    const lang = req.locale.id || 'en'
+    const translationsJSON = JSON.stringify(TRANSLATIONS[lang] || {})
+
+    const privacyPolicyMD = loadLocaleRawFile('privacy-policy.md', lang)
+
+    const userMockData = {
+      firstName: req.user?.firstName,
+      lastName: req.user?.lastName
+    }
+
+    res.render('eula', {
+      config: CONFIG,
+      userData: userMockData,
```

Comment:
Just to confirm but is it necessary pass the `userData` in this template? I don't see any reference about it (I don't know how this works exactly 🙏 ).

Reply:
Good catch, @Tansito - I'll remove it and resubmit the PR.